Source/WebKit/DerivedSources.make

```diff
@@ -225,6 +225,7 @@ MESSAGE_RECEIVERS = \
 	WebProcess/WebCoreSupport/WebBroadcastChannelRegistry \
 	WebProcess/WebCoreSupport/WebDeviceOrientationUpdateProvider \
 	WebProcess/WebCoreSupport/WebFileSystemStorageConnection \
+	WebProcess/WebCoreSupport/WebPermissionController \
```

Comment:
An entry like this should be added in the `Source/WebKit/CMakeLists.txt`, in the `WebCoreSupport` block around lines 321-324 to trigger the Messages derived sources generation for `WebPermissionController`, no?

The PR as it is is failing in both GTK/WPE, leaving the build in a inconsistent state: https://ews-build.webkit.org/#/builders/36/builds/80200

Reply:
Yes, you're right. It looks like I added this entry in `WebKit/DerivedSources.make` but forgot to add it to `WebKit/CMakeLists.txt`. I'll add it there as well. Thank you!